optimum/onnxruntime/trainer.py

```diff
@@ -1455,24 +1455,23 @@ def _export(
                 self.model.to("cpu")
             model = unwrap_model(self.model)
 
-        _, model_onnx_config = FeaturesManager.check_supported_model_or_raise(model, feature=self.feature)
-        onnx_config = model_onnx_config(model.config)
-        opset = onnx_config.default_onnx_opset if opset is None else opset
+        model_type = model.config.model_type.replace("_", "-")
+        model_name = getattr(model, "name", None)
```

Comment:
I saw this also in ORTModels' export, why didn't we make it something inside ` TasksManager.get_exporter_config_constructor`?

Reply:
Yes, I think we can do that. Updated with the change.  

cc: @michaelbenayoun if there is needs to be reverted

Reply:
I guess it is good for now (I do not have any potential issue in mind)